.github/workflows/all.testWithRpc.yml

```diff
@@ -6,7 +6,9 @@ on:
 jobs:
   test-node:
     name: testing
-    runs-on: token-studio-linux-medium
+#    runs-on: [self-hosted, Linux, medium, ephemeral]
+    runs-on: ubuntu-latest
```

Comment:
Should be token-studio-linux-medium, it was updated recently by another PR

Reply:
Ok! done

Reply:
with token-studio-linux-medium test dont pass. With ubuntu-latest all test pass, could we keep ubuntu-latest?